datahub-web-react/src/graphql/contract.graphql

```diff
@@ -0,0 +1,53 @@
+fragment contractAssertionParameters on AssertionStdParameters {
+    value {
+        value
+        type
+    }
+    minValue {
+        value
+        type
+    }
+    maxValue {
+        value
+        type
+    }
+}
+
+fragment dataContractDetails on DataContract {
+    urn
+    type
+    properties {
+        freshness {
+            assertion {
+                ...assertionDetailsWithRunEvents
```

Comment:
we should copy this into SaaS too if the SaaS has no need for monitor info on data contracts

Reply:
Done